Automattic/wp-calypso
# Stats: Adding missing icons

Related to https://github.com/Automattic/red-team/issues/83

## Proposed Changes

* adding icons needed to update Stats module empty states
* updating empty Top posts string
* moving empty class name

## Why are these changes being made?


* resources need to finish the current project

## Testing Instructions



* open the live branch
* apply `?flags=stats/empty-module-traffic`
* verify that the cards match the screenshot

![SCR-20240704-jnld](https://github.com/Automattic/wp-calypso/assets/112354940/c6e72236-6720-4b25-be98-1754f57a66db)

![SCR-20240704-jnnk](https://github.com/Automattic/wp-calypso/assets/112354940/aab3d1ac-934e-40f5-8313-75ce6f096013)


## Pre-merge Checklist
